tasks/task05/Userfactory.py

```diff
@@ -0,0 +1,35 @@
+def create_user(*identity, age=42, **kwargs):
```

Comment:
for clarity it's better to name all the positional arguments and not to unpack list.

also in this case I can pass unlimited number of arguments, which in most cases is not what you would like to have as an function input.